spyder/plugins/switcher/widgets/proxymodel.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""Switcher Proxy Model."""
+
+# Third party imports
+from qtpy.QtCore import (QSortFilterProxyModel, Qt)
+
+# Local imports
+from spyder.config.base import _
+
+class SwitcherProxyModel(QSortFilterProxyModel):
```

Comment:
```suggestion

class SwitcherProxyModel(QSortFilterProxyModel):
```

We tend to leave two lines between the imports section and what's next.